assets/css/_card.scss

```diff
@@ -22,16 +16,27 @@
 }
 
 .m-card__left {
-  display: flex;
-  flex-direction: column;
   flex-grow: 1;
   position: relative;
+  display: grid;
+  grid-template-areas: "icon main-contents";
+  grid-template-columns: min-content 1fr;
 
   &:last-child {
     padding-right: 0.3125rem;
   }
 }
 
+.m-card__icon {
+  grid-area: icon;
+}
+
+.m-card__left-content {
+  display: flex;
+  flex-direction: column;
+  grid-area: main-contents;
+}
+
```

Comment:
suggstion(non-blocking): you're using this grid for 1d layout, but then putting a flexbox in the 1d layout in the oposite axis it seems,  I don't think both are needed here and I think you should consider consolodating down to using either grid or flexbox. I think you could do this all with grid though, as grid's meant for 2d layouts.